elementor/elementor
# Internal: Conversion Center - Address Feedback and Refactor class for Pro

## PR Checklist


## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Address Feedback related to Link In bio
* Refactor rendering classes to allow better extendability

## Description
An explanation of what is done in this PR

* Address Feedback related to Link In bio
* Refactor rendering classes to allow better extendability

## Test instructions
This PR can be tested by following these steps:

* Add the Link In bio Widget and verify that things have been addressed

## Quality assurance


Fixes #
